pkg/maintainer/spv/deposit_sweep_proof.go

```diff
@@ -0,0 +1,190 @@
+package spv
+
+import (
+	"fmt"
+
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/chain"
+	"github.com/keep-network/keep-core/pkg/tbtc"
+)
+
+// SubmitDepositSweepProof prepares deposit sweep proof for the given
+// transaction and submits it to the onchain contract.
+func SubmitDepositSweepProof(
+	transactionHash bitcoin.Hash,
+	requiredConfirmations uint,
+	btcChain bitcoin.Chain,
+	tbtcChain tbtc.Chain,
```

Comment:
There are only two functions used from `tbtc.Chain`: `SubmitDepositSweepProofWithReimbursement` and `GetDepositRequest`. Instead of using `tbtc.Chain`, can we introduce `pkg/maintainer/spv/chain.go` (see `pkg/maintainer/btcdiff/chain.go` and `pkg/maintainer/wallet/chain.go` with only those two functions? I hope we can then avoid some of the changes we had to make in `*_test.go` files in the unrelated packages.

Reply:
Done in 03a6748e318fcabeed2aa6cd50614320906a7d9f.